client/src/App.js

```diff
@@ -3,23 +3,51 @@ import ChatPage from './pages/ChatPage.js';
 import LoginPage from "./pages/LoginPage.js";
 import "./App.css";
 import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import '../node_modules/bootstrap/dist/js/bootstrap.bundle.min.js';
+
+const url = "http://localhost:5000/";
+//fetch(url + "login/" + localStorage.getItem('login')))
```

Comment:
You should just delete all the stuff you have commented

Reply:
This has now been fixed